### PR TITLE
Added filters in the webhooks

### PR DESCRIPTION
**Pull request by rounak610**
### Description

Added filters in the webhooks

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have read the contributing guide and my code conforms to the guidelines.
- [ ] I have documented my changes clearly and comprehensively.
- [ ] I have added the required tests.

### CODE REVIEW

File: migrations/versions/40affbf3022b_add_filter_colume_in_webhooks.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+revision = '40affbf3022b'
+down_revision = '5d5f801f28e7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index('ix_agent_schedule_agent_id', table_name='agent_schedule')
+    op.drop_index('ix_agent_schedule_expiry_date', table_name='agent_schedule')
```
Comment (Fluder-Paradyne) [inline migrations/versions/40affbf3022b_add_filter_colume_in_webhooks.py:22]: remove all drop index

why is there alter_columns ?

if alter_columns were auto generated remove them also

Comment (rounak610) [reply migrations/versions/40affbf3022b_add_filter_colume_in_webhooks.py:22]: I just used these two commands:
docker compose exec -it backend bash
 then
alembic revision --autogenerate -m "add filter colume in webhooks"
and then this file was automatically generated.

Comment (rounak610) [reply migrations/versions/40affbf3022b_add_filter_colume_in_webhooks.py]: Made the changes as you suggested

File: superagi/controllers/webhook.py [hunk 2]
```diff
@@ -52,9 +62,55 @@ def create_webhook(webhook: WebHookIn, Authorize: AuthJWT = Depends(check_auth),
             HTTPException (Status Code=404): If the associated project is not found.
     """
     db_webhook = Webhooks(name=webhook.name, url=webhook.url, headers=webhook.headers, org_id=organisation.id,
-                          is_deleted=False)
+                          is_deleted=False, filters=webhook.filters)
     db.session.add(db_webhook)
     db.session.commit()
     db.session.flush()
-
+    print (db_webhook)
```
[the remark this answers is not in the document]
Comment (rounak610) [reply superagi/controllers/webhook.py]: done

**Review by nborthy — APPROVED** [review]

**Review by Fluder-Paradyne — APPROVED** [review]

**Pull request merged**